spec.proto

```diff
@@ -0,0 +1,55 @@
+/**
+ * CloudEvent Protobuf Format
+ *
+ * - Required context attributes are explicity represented.
+ * - Optional and Extension context attributes are carried in a map structure.
+ * - Data may be represented as binary, text, or protobuf messages.
+ */
+
+syntax = "proto3";
+
+package io.cloudevents.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_package = "io.cloudevents.v1.proto";
+
+message CloudEvent {
+
+  // -- CloudEvent Context Attributes
```

Comment:
We should revisit all the comments. I'd expect a comment for every field and message.

Reply:
Ack ... Will address...

Reply:
Ideally it would be good to get this done before it's merged, but I'm okay for us to do it later.